src/planner/insert_plan.cpp

```diff
@@ -66,6 +66,11 @@ InsertPlan::InsertPlan(
           is_prepared_stmt = true;
         }
       }
+      // for remaining columns, insert defaults
+      for (uint32_t column_id = values.size(); column_id != schema_col_count;
+           ++column_id) {
+        SetDefaultValue(column_id);
+      }
```

Comment:
Good. Thank you. I missed this.
No test added for this though? 
I will add one in the JUnit tests, since I'm working on those now.

Reply:
Cool. So should I add a test in insert_sql_test or not?

Reply:
No. I would not worry. I've got it covered. Will be in a PR as soon as I have it working as desired on Travis and Jenkins.